client/state/media/actions.js

```diff
@@ -6,21 +6,61 @@ import { castArray } from 'lodash';
 /**
  * Internal dependencies
  */
-import { MEDIA_DELETE, MEDIA_RECEIVE } from 'state/action-types';
+import {
+	MEDIA_DELETE,
+	MEDIA_RECEIVE,
+	MEDIA_REQUEST,
+	MEDIA_REQUESTING } from 'state/action-types';
 
 /**
  * Returns an action object used in signalling that media item(s) for the site
  * have been received.
  *
  * @param  {Number}         siteId Site ID
  * @param  {(Array|Object)} media  Media item(s) received
+ * @param  {Number}         found  Number of found media
+ * @param  {Object}         query  Query Object
  * @return {Object}                Action object
  */
-export function receiveMedia( siteId, media ) {
+export function receiveMedia( siteId, media, found, query ) {
```

Comment:
Failing test on this action creator, likely not expecting the additional properties.